integr8ly/cloud-resource-operator
# fix in cluster postgres provisioning

currently we do not give the correct permissions to the user we
create in the postgres instance for openshift deployments. in rhmi
codeready workspaces requires the user to be a superuser.

this defaults the created user to be a superuser.

verification:
- have an openshift 4 cluster running
- have this operator running
- have the rhmi operator running from austincunningham:INTLY-3146
- create an installation cr

```
apiVersion: integreatly.org/v1alpha1
kind: Installation
metadata:
  name: example-installation
spec:
  type: workshop
  namespacePrefix: integreatly-
  routingSubdomain: apps.example.com
  masterUrl: http://console.apps.example.com
  selfSignedCerts: true
  useExternalResources: true
```

- ensure codeready comes up
- ensure codeready has no self-deployed database
- ensure codeready migrations complete successfully
